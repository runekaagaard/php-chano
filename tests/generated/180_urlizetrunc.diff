001- <a href="http://31characteruri.com/test/" rel="nofollow">...</a>
001+ Fatal error: Call to undefined method DtlIter::urlizetrunc() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/180_urlizetrunc.php on line 4