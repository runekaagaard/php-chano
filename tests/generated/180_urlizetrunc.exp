<a href="http://31characteruri.com/test/" rel="nofollow">...</a>