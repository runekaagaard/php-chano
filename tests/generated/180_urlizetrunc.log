
---- EXPECTED OUTPUT
<a href="http://31characteruri.com/test/" rel="nofollow">...</a>
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::urlizetrunc() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/180_urlizetrunc.php on line 4
---- FAILED
