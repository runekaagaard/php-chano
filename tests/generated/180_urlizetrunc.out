Fatal error: Call to undefined method DtlIter::urlizetrunc() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/180_urlizetrunc.php on line 4